cloudify_gcp/utils.py

```diff
@@ -236,6 +256,8 @@ def throw_cloudify_exceptions(func):
     def _decorator(*args, **kwargs):
         try:
             return func(*args, **kwargs)
+        except (RecoverableError, NonRecoverableError) as e:
+            raise e
```

Comment:
@GhalebKhaled Do we need to re-raise those two exceptions? because If even if we do not catch them they will raise as they are, so I think adding this except does not help because  `RecoverableError, NonRecoverableError` will be raised at the end of the day

Reply:
The logic below is wrapping the rest of error into `NonRecoverableError`
https://github.com/cloudify-cosmo/cloudify-gcp-plugin/pull/86/files#diff-603868481f6f0761d0d86dd368222d45R273
So I added a first check for `RecoverableError, NonRecoverableError` and raise them as they are, since 
- I don't want to wrap `RecoverableError` into `NonRecoverableError`
- I don't want to wrap `NonRecoverableError` into `NonRecoverableError` again

Reply:
Ok